ffmpeg/ffmpeg.go

```diff
@@ -399,9 +445,11 @@ func (t *Transcoder) Transcode(input *TranscodeOptionsIn, ps []TranscodeOptions)
 			opts: newAVOpts(p.AudioEncoder.Opts),
 		}
 		vfilt := C.CString(filters)
+		sfilt := C.CString(signfilter)
```

Comment:
This line and its corresponding free-ing call can be moved inside `if p.CalcSign {` block.